lib/demos/reference/motion_demo_container_transition.dart

```diff
@@ -129,8 +129,19 @@ class _OpenContainerTransformDemoState
             key: _scaffoldKey,
             appBar: AppBar(
               automaticallyImplyLeading: false,
-              title: Text(
-                localizations.demoContainerTransformTitle,
+              title: Column(
+                children: [
+                  Text(
+                    localizations.demoContainerTransformTitle,
+                  ),
+                  Text(
+                    '(${localizations.demoContainerTransformDemoInstructions})',
+                    style: Theme.of(context)
+                        .textTheme
+                        .subtitle2
+                        .copyWith(color: Colors.white),
```

Comment:
Since the title is already white, do you also need to explicitly set the subtitle to white?

If not, I'd remove that throughout this PR

Reply:
The default subtitle color was black.